python/hail/plot/plots.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from math import log, isnan
+from bokeh.io import show
+from bokeh.plotting import figure
+from bokeh.layouts import gridplot
+from bokeh.models import Span, ColumnDataSource, CategoricalColorMapper
+from bokeh.palettes import Category10
+
+
+def histogram(data, label=None, title=None, ylabel='Frequency'):
```

Comment:
OK, we want two things here:

1. We want type to typecheck `histogram`.  That will looks something like:

```
@typecheck(data=sequenceof(numeric), label=nullable(str), title=nullable(str), ylabel=nullable(str))
def histogram(data, ...):
    ...
```

Then, when histogram is called, it will make sure each argument has the required type, otherwise throw a nice error message.

2. Documentation.  I'd look at functions in table.py or matrixtable.py for the syntax.  Basically, you want a short description (Plot a histogram.) and the types and short descriptions of the parameters.

Reply:
@cseed for the typecheck, the data parameter is currently of type `hail.utils.struct.Struct`, which Tim says there isn't a typecheck for. Should I hold off on the typecheck until the version in which we use table fields?